cEP-0025.md

```diff
@@ -0,0 +1,306 @@
+# Integration of pyflakes-enhanced-AST into coala
+
+| Metadata |                                                 |
+| -------- | ----------------------------------------------- |
+| cEP      | 22                                              |
+| Version  | 1.0                                             |
+| Title    | Integration of pyflakes-enhanced-AST into coala |
+| Authors  | Ankit Joshi <mailto:ajankit2304@gmail.com>      |
+| Status   | Proposed                                        |
+| Type     | Feature                                         |
+
+## Abstract
+
+This document describes how to create a metabear to integrate
+pyflakes-enhanced-AST into coala and demonstrates its use by
+creating a simple dependent bear.
+
+## Introduction
+
+Flake8 is the most commonly used linter framework when it comes to static
+code analysis for python. It wraps around the tools pyflakes,
+pycodestyle and mccabe scripts. These tools are implemented on different
+interfaces and flake8 combines their potential.
+
+It even provides with a plugin mechanism. Additional linters are
+supported using the flake8 plugin system, and may be Python AST
```

Comment:
replace `AST` here , as it is actually the CPython `ast` standard library package which is used.
Make that very clear here, which helps show that the pyflakes AST is quite different.